### PR TITLE
Sauce Labs Browser Matrix big–badge

### PR DESCRIPTION
**Pull request by mightyiam**
This is widely used.

Not yet tested. Just let me test it first, please.

### CODE REVIEW

**Comment by mightyiam** [comment]
It involves some build procedure that I'm not familiar with. Can we get this in without me testing?

**Comment by mightyiam** [comment]
Or, can you please give me a pointer on how I can get this built?

File: src/lib/badge-util.coffee [hunk 1]
```diff
@@ -1,6 +1,19 @@
 # Define
 module.exports = badgeUtil =
 	miscBadges:
+		# Get Sauce Labs Browser Matrix
+		sauceBM: (opts={}) ->
```
Comment (RobLoach) [inline src/lib/badge-util.coffee]: Could we make this lowercase this? Make it match the other standards?

**Comment by RobLoach** [comment]
Might also want to update the docs :+1: .

**Comment by mightyiam** [comment]
Thanks, Rob. I got tangled up in the cake/docco net. I saw a big Cakefile and it is just too much for me.

I tried running `$docco -o docs/ src/lib/*` and it did compile but it made a lot of changes.

I did that lowercase fix you requested. Can you take care of the rest for us, please?

**Comment by mightyiam** [comment]
Any love?

**Comment by mightyiam** [comment]
Thank you.

BTW: I've switched over to https://github.com/jmanero/auto-package and https://github.com/assemble/verb.

**Pull request merged**